src/chatdlg.cpp

```diff
@@ -67,6 +67,10 @@ CChatDlg::CChatDlg ( QWidget* parent ) :
         SLOT ( OnClearChatHistory() ), QKeySequence ( Qt::CTRL + Qt::Key_E ) );
 
     pMenu->addMenu ( pEditMenu );
+#if defined ( Q_OS_IOS )
+    QAction *action = pMenu->addAction (tr ( "&Close" ));
```

Comment:
Good idea. Probably also something which should be part of Android. There's an issue on this, I think.